designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -113,23 +107,17 @@ Lists the package references for a project or solution.
 ```shell
 > dotnet list package
 
-The following sources were used:
-   nuget.org - https://api.nuget.org/v3/index.json
-   Local - C:\NuGet\NuGetLocal
-
 Project 'MySampleProject' has the following package references
    [netcoreapp2.1]:
    Top-level Package               Requested   Resolved
    > Microsoft.ML                  0.11.0      0.11.0
    > Microsoft.NETCore.App   (A)   [2.1.0, )   2.1.0
-   > My.Outdated.Package           0.1.0       0.1.0 (O)
-   > My.Deprecated.Package         1.0.0       1.0.0 (D)
-   > My.Vulnerable.Package         1.0.1       1.0.1 (V)
+   > My.Outdated.Package           0.1.0       0.1.0
+   > My.Deprecated.Package         1.0.0       1.0.0
+   > My.Vulnerable.Package         1.0.1       1.0.1
 
+ > To see transitive packages, add the `--include-transitive` option.
```

Comment:
Should we include hints for the `--vulnerable`, `--outdated`, and `--deprecated` options? Something like:

```suggestion
 > To see transitive packages, add the `--include-transitive` option.
 > To see outdated packages, add the `--outdated` option.
 > To see deprecated packages, add the `--deprecated` option.
 > To see vulnerable packages, add the `--vulnerable` option.
```

Note that this may cause users to try and mix and match options like `--deprecated --vulnerable`. If this is a concern, consider adding language warning against this.

Reply:
Another thought could be to collapse all of this into:
- For details on including transitive dependencies or to view outdated/deprecated/vulnerable reports, see `dotnet list package --help`

Reply:
fwiw, I like @drewgillies's latest suggestion.

Reply:
Yup I like @drewgillies's suggestion too!

Reply:
Per @JonDouglas's comment above, we'll remove footers for now and let customers use `--help` for the full help experience.